3i-nodejs/recipes/setup-pm2.rb

```diff
@@ -22,15 +22,3 @@
   user 'ubuntu'
   environment ({'HOME' => '/home/ubuntu'})
 end
-
-execute "setup-keymetrics-server-monitoring" do
```

Comment:
We can bring this back into staging if we want to see if their new changes fixed the CPU alert, correct?

Reply:
Yep! Though I'm guessing PM2 plus may have some analog they release in the future that will be better supported 🤞